UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -838,16 +838,11 @@ <h2>Discovering accessible content</h2>
 
 		<section id="localization">
 			<h3>Localization</h3>
-
-<p>In these guidelines, we have used  simple language  to  communicate the meaning of the metadata values. Many people have contributed to  development  of the words and phrases  we have selected. We intend to provide a mechanism for the publishing community to provide translations that localizes the strings for equally effective communication in many languages. We understand that simply translating the strings is not enough; the subtle meaning of the words and phrases of accessibility concepts must be localized for maximum understanding.
-</p>
-
-<p>At this time, we have not decided on the technical mechanism for providing   the localization. Our next steps are to work on the techniques and identify the strings in English that are associated with the metadata values. We will probably look at localizing for French and Italian before deciding on the technical approach.
-</p>
-
-<p>We want to alert the community that the translation and localization of the information to be communicated is under development and will be delivered as soon as possible.
-</p>
-			<p> The full approach TBD</p>
+				<p>In these guidelines, we have used  simple language  to  communicate the meaning of the metadata values. Many people have contributed to  development  of the words and phrases  we have selected. We intend to provide a mechanism for the publishing community to provide translations that localizes the strings for equally effective communication in many languages. We understand that simply translating the strings is not enough; the subtle meaning of the words and phrases of accessibility concepts must be localized for maximum understanding.</p>
+				<p>For users looking for books, a similar UX experience is crucial to not add cognitive charge to the already complex task of understanding what features are included in the book and which way to access to it have been thoughts.</p>
```

Comment:
This sounds a little awkward.

For users seeking books, a seamless UX (user experience) is essential to avoid adding cognitive load to the already complex task of understanding the included features and accessing the book.